lib/schemaUtils.js

```diff
@@ -1873,13 +1873,25 @@ module.exports = {
         }
       }
 
-      bodyData = this.convertToPmBodyData(contentObj[bodyType], requestType, bodyType,
-        PARAMETER_SOURCE.REQUEST, options.indentCharacter, components, options, schemaCache);
+      if (
+        bodyType &&
+        contentObj[bodyType].hasOwnProperty('schema') &&
+        contentObj[bodyType].schema.type === 'string' &&
```

Comment:
This sentence can break if `contentObj[bodyType].schema` is set to `null`. In above sentence check with following.
```
!_.isEmpty(_.get(contentObj, [bodyType, 'schema']))
```